tests/test_project.py

```diff
@@ -64,17 +74,18 @@ def test_project_repr(client):
 
 def test_create_project_success(client):
     token = login(client, ADMIN_USERNAME, ADMIN_PWD)
-    response = client.post(PROJECT_ENDPOINT,
-                           json={'project_name': SUCCESS_PROJECT_NAME},
-                           headers={'Authorization': 'Bearer %s' % token})
+    response = client.post(
+        PROJECT_ENDPOINT,
+        json={'project_name': SUCCESS_PROJECT_NAME},
+        headers={'Authorization': 'Bearer %s' % token})
     assert response.status_code == 201
     res = response.get_json()
     assert 'project_id' in res
 
 
 def test_create_project_not_logged_in(client):
-    response = client.post(PROJECT_ENDPOINT,
-                           json={'project_name': FAIL_PROJECT_NAME})
+    response = client.post(
```

Comment:
previously, you had one parameter per line, but here you have two parameters in the same line

Reply:
The coding style as dictated by the Google Python Style Guide (http://google.github.io/styleguide/pyguide.html) says that there must not be more than 80 chars per line. However, it's true that it's not consistent with the other lines in the file, so we will change it.